litellm/llms/bedrock_httpx.py

```diff
@@ -238,10 +240,41 @@ def get_credentials(
             aws_session_name,
             aws_profile_name,
             aws_role_name,
+            aws_web_identity_token,
         ) = params_to_check
 
         ### CHECK STS ###
-        if aws_role_name is not None and aws_session_name is not None:
+        if aws_web_identity_token is not None and aws_role_name is not None and aws_session_name is not None:
+            oidc_token = get_secret(aws_web_identity_token)
```

Comment:
@Manouchehri these would also be sync http requests for async calls. I view fixing this - https://github.com/BerriAI/litellm/issues/3858#issuecomment-2133785581 as a precursor to this PR.

Reply:
what do you mean. Your calling 'get_secret' inside of which you use HTTPHandler

The request is to use AsyncHTTPHandler instead.